aclk/README.md

```diff
@@ -1,10 +1,89 @@
 <!--
 ---
 title: "Agent-cloud link (ACLK)"
+description: "The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. 
+The ACLK uses MQTT over secure websockets to create, persist, and encrypt the connect from end-to-end, and enable the 
+features found in Netdata Cloud."
+date: 2020-03-18
 custom_edit_url: https://github.com/netdata/netdata/edit/master/aclk/README.md
 ---
 -->
 
 # Agent-cloud link (ACLK)
 
-This is the agent cloud link (ACLK) information file
\ No newline at end of file
+The agent-cloud link (ACLK) is the mechanism responsible for connecting a Netdata agent to Netdata Cloud. The ACLK uses
+[MQTT](https://en.wikipedia.org/wiki/MQTT) over secure websockets to create, persist, and encrypt the connect from
+end-to-end, and enable the features found in Netdata Cloud.
+
+The ACLK is a vital component of the entirely redesigned and re-architected Netdata Cloud.
+
+_No data is exchanged with Netdata Cloud until you claim a node._ By claiming a node, you opt-in to sending data from
+your agent to Netdata Cloud via the ACLK. Read our [claiming documentation](../claim/README.md) for details on node
+claiming.
+
+## Enable and configure the ACLK
+
+The ACLK is enabled by default if its prerequisites installed correctly, and the correct configuration is already set in
+your agent's `netdata.conf` file.
+
+> 🎆 Needed: Any other configuration settings?
+
+```conf
+[cloud]
+    cloud base url = https://up.netdata.cloud
+```
+
+### Proxy configuration
+
+If your Netdata agent uses a proxy to reach the outside internet, you must configure a SOCKS5 proxy using one of the
+following methods.
+
+> 🎆 Needed: Configuration options/env variables for proxy.
```

Comment:
I put mine near the top of `/usr/sbin/netdata-claim.sh`, and it works:

```
ryan@ryan-VirtualBox:~$ head /usr/sbin/netdata-claim.sh
#!/usr/bin/env bash
# netdata
# real-time performance and health monitoring, done right!
# (C) 2017 Costa Tsaousis <costa@tsaousis.gr>
# SPDX-License-Identifier: GPL-3.0-or-later

export https_proxy=socks5://127.0.0.1:11081

# Exit code: 0 - Success
# Exit code: 1 - Unknown argument
```

Reply:
I haven't tried with any kind of authentication, though.

Reply:
You can do that with the -proxy switch instead (its in the README.md in the claim directory) or by exporting https_proxy in the shell (so that curl picks it up).